packages/core/src/kernelCoreTemplate.js

```diff
@@ -72,6 +72,9 @@
     const rootPackageName = '$root$'
     const rootPackageCompartment = createRootPackageCompartment(globalRef)
 
+    // scuttle globalThis right after we used it to create the root package compartment
+    scuttleGlobalThis()
```

Comment:
Feels like we'd want this to be configurable in the final version

Reply:
https://github.com/LavaMoat/LavaMoat/pull/376/files
(ignore 'runtime.js` was added by mistake)

Reply:
ik